src/common/POLFields/FieldHolding/getCallNumber.js

```diff
@@ -0,0 +1,3 @@
+export const getCallNumber = (callNumber = '', callNumberPrefix = '', callNumberSuffix = '') => (
+  `${callNumberPrefix}${callNumber}${callNumberSuffix}`.trim()
```

Comment:
you don't need this function at all, just use ` `${callNumberPrefix}${callNumber}${callNumberSuffix}`.trim()` is place

Reply:
callNumberPrefix, callNumber, callNumberSuffix are not required and could be undefined

Reply:
if you want move logic, please move whole logic to get holding options